fem/libceed/operator.cpp

```diff
@@ -0,0 +1,359 @@
+// Copyright (c) 2010-2020, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+
+#include "operator.hpp"
+
+#include "../../config/config.hpp"
+#include "../../linalg/vector.hpp"
+#include "../fespace.hpp"
+#include "../../general/forall.hpp"
+
+#ifdef MFEM_USE_CEED
+
+namespace mfem
+{
+
+void MFEMCeedOperator::Mult(const Vector &x, Vector &y) const
```

Comment:
The naming should be revised in view of Yohanns PR introducing the `CEED::` namespace